documentation/fx-implementation/README.md

```diff
@@ -0,0 +1,48 @@
+# FX Implementation
+
+## Proof of Concept (PoC) Implementation for Payer-Side Currency Conversion (Happy Path Only)
```

Comment:
Does this still hold?

Reply:
Hey Sam. Yes currently only payer side conversion is fully supported.

Reply:
Thanks @kleyow , also Is it still a PoC implementation?

Reply:
@elnyry-sam-k Personally, I'm not sure.